model/task_lifecycle.go

```diff
@@ -628,6 +628,34 @@ func doBisectStepback(ctx context.Context, t *task.Task) error {
 	if nextTask == nil {
 		return errors.Errorf("midway task could not be found for tasks '%s' '%s'", s.LastFailingStepbackTaskId, s.LastPassingStepbackTaskId)
 	}
+	// Log our next task to our last passing and last failing.
+	for _, id := range []string{s.LastFailingStepbackTaskId, s.LastPassingStepbackTaskId} {
+		if t, err := task.FindOneId(id); err != nil {
```

Comment:
nit: This might do well with the logic from 633 --> 656 extracted into a helper called `setStepbackTaskId` or something like that and calling it on `s.LastFailingStepbackTaskId` and `s.LastPassingStepbackTaskId`